### ISSUE TITLE
Finish migrating to naive-ui

### ISSUE THREAD

**Issue by tobiasdiez** [JabRef/JabRefOnline#1375]
Goal: remove tailwind-vue dep completely, and migrate to naive-ui instead

**Comment by mohit038** [comment]
I think [vuetify](https://vuetifyjs.com/en/) has more UI components and is a more mature UI lib for Vue.
I there any specific reason for using this library?

**Comment by tobiasdiez** [comment]
Yes, vuetify is nice as well. I'm just not a big fan of material design and vuetify doesn't play nicely along with tailwind, which was one of the biggest reasons for not using it.
